src/component/variantPage/GnomadData.tsx

```diff
@@ -0,0 +1,265 @@
+import * as React from 'react';
```

Comment:
Is this the modified version of `GnomadFrequency` from `react-mutation-mapper`? Can you mark the modified parts (by adding comments)? It would make it easier to add those changes back to `react-mutation-mapper` later.

Reply:
@onursumer I marked modified parts with `// MODIFIED`